app/src/main/java/com/example/vybe/AddEditVibeActivity.java

```diff
@@ -304,4 +310,27 @@ public String formatDateTime(LocalDateTime dateTime) {
         return dateTime.format(DateTimeFormatter.ofPattern("MMM dd yyyy, hh:mm a"));
     }
 
+    /**
+     * This will load an image from Firebase Storage into an ImageView
+     * @param imageView
+     *      The destination ImageView
+     * @param imagePath
+     *      Path to the image in Firebase Storage
+     */
+    public void loadImageFirebase(ImageView imageView, String imagePath){
+        StorageReference storageRef = FirebaseStorage.getInstance().getReference();
+
+        // Get the path to the image
+        StorageReference imageRef = storageRef.child(imagePath);
+
+        // Get the download URL for Glide
+        imageRef.getDownloadUrl().addOnSuccessListener(new OnSuccessListener<Uri>() {
+            @Override
+            public void onSuccess(Uri uri) {
+                Glide.with(getApplicationContext() /* context */)
```

Comment:
What up with the `/* context */` comment

Reply:
Removed